### PR TITLE
Fix mistake in docs/contributing/test.md running specific unit test #38096

### LINKED ISSUE #38096: Possible mistake in contributing/test.md regarding running specific unit test
**Description**

**Steps to reproduce the issue:**
1. Go through the [doc](https://github.com/moby/moby/blob/master/docs/contributing/test.md#run-unit-tests)

**Describe the results you received:**
After issuing:
```
$ TESTDIRS='opts' TESTFLAGS='-test.run ^TestValidateIPAddress$' make test-unit
```
from the test documents, I got:
```
can't load package: package opts: cannot find package "opts" in any of:
        /usr/local/go/src/opts (from $GOROOT)
        /go/src/opts (from $GOPATH)
make: *** [Makefile:161: test-unit] Error 1
```

**Describe the results you expected:**
However I managed to get it to work with:
```
$ TESTDIRS='github.com/docker/docker/opts' TESTFLAGS='-test.run $^TestValidateIPAddress$' make test-unit
```
Note the full path `github.com/docker/docker/opts`. Which is expected since
```
    go test "${BUILDFLAGS[@]}" \
        -cover \
        -coverprofile=profile.out \
        -covermode=atomic \
        $TESTFLAGS \
        "${pkg}"
```
from `hack/test/unit` should probably not be issuing `go test opts`.
This resulted in:
```
ok      github.com/docker/docker/opts   0.028s  coverage: 0.0% of statements [no tests to run]                                                                           
```

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `docker version`:**

```
Client:
 Version:           18.06.1-ce
 API version:       1.38
 Go version:        go1.11
 Git commit:        e68fc7a215
 Built:             Fri Sep  7 11:26:59 2018
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.1-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.11
  Git commit:       e68fc7a215
  Built:            Fri Sep  7 11:26:11 2018
  OS/Arch:          linux/amd64
  Experimental:     false
```

**Output of `docker info`:**

```
Containers: 1
 Running: 1
 Paused: 0
 Stopped: 0
Images: 83
Server Version: 18.06.1-ce
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: false
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 468a545b9edcd5932818eb9de8e72413e616e86e
runc version: 69663f0bd4b60df09991c08812a60108003fa340
init version: fec3683
Security Options:
 seccomp
  Profile: default
Kernel Version: 4.14.70-1-lts
Operating System: Arch Linux
OSType: linux
Architecture: x86_64
CPUs: 4
Total Memory: 7.679GiB
Name: colonelthink
ID: J6N7:37O7:PQEO:VI7Y:NQAQ:JFJ5:77WR:FFF7:7CF7:MUIC:MZF2:ZVTZ
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Labels:
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

### PR DESCRIPTION
**Pull request by jaswdr**
**- What I did**
Fixes #38096

**- How I did it**
Updating the docs with more specific path when running specific unit tests

**- How to verify it**
Running the code bellow.

```sh
$ TESTDIRS='github.com/docker/docker/opts' TESTFLAGS='-test.run $^TestValidateIPAddress$' make test-unit
```

**- Description for the changelog**
Corrected running specific unit tests script in docs

**- A picture of a cute animal (not mandatory but encouraged)**

image

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]

File: docs/contributing/test.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ flag's value is passed as arguments to the `go test` command. For example, from
 your local host you can run the `TestValidateIPAddress` test with this command:
 
 ```bash
-$ TESTFLAGS='-test.run ^TestValidateIPAddress$' make test-unit
+$ TESTDIRS='github.com/docker/docker/opts' TESTFLAGS='-test.run $^TestValidateIPAddress$' make test-unit
```
Comment (thaJeztah) [inline docs/contributing/test.md]: Shouldn't this be updated on line 151 instead?

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: docs/contributing/test.md [hunk 2]
```diff
@@ -148,7 +148,7 @@ You can use the `TESTDIRS` environment variable to run unit tests for
 a single package.
 
 ```bash
-$ TESTDIRS='opts' make test-unit
+$ TESTDIRS='github.com/docker/docker/opts' TESTFLAGS='-test.run $^TestValidateIPAddress$' make test-unit
```
Comment (thaJeztah) [inline docs/contributing/test.md]: This section talks about the `TESTDIRS` environment variable, so I guess this should be;

```suggestion
$ TESTDIRS='github.com/docker/docker/opts' make test-unit
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]
left a comment; also could you squash your commits, so that there's only a single commit in this PR?

**Comment by jaswdr** [comment]
@thaJeztah I think now is ok

**Review by thaJeztah — APPROVED** [review]
LGTM

thank you so much!

**Pull request merged**